### PR TITLE
[dv,usbdev] Transition USBDEV to V2S

### PR DESCRIPTION
**Pull request by alees24**

### CODE REVIEW

File: hw/ip/usbdev/data/usbdev.hjson [hunk 1]
```diff
@@ -21,7 +21,8 @@
   version:            "2.0.0",
   life_stage:         "L1",
   design_stage:       "D2S",
-  verification_stage: "V1",
+  verification_stage: "V2S",
+  commit_id:          "3f85737335e12afcff4e37d163c2849ab2e49846"
```
Comment (andreaskurth) [inline hw/ip/usbdev/data/usbdev.hjson:25]:
```suggestion
```
We no longer use the `commit_id` field, instead we would obtain the Git commit hash at signoff via tooling. Could you please remove that?

Comment (alees24) [reply hw/ip/usbdev/data/usbdev.hjson]: "When in doubt look at the UART." Does this want to be changed? -> https://github.com/lowRISC/opentitan/blob/master/hw/ip/uart/data/uart.hjson

Comment (andreaskurth) [reply hw/ip/usbdev/data/usbdev.hjson]: Yes, at some point :-)

**Review by andreaskurth — APPROVED** [review]

**Comment by andreaskurth** [comment]
CHANGE AUTHORIZED: hw/ip/usbdev/data/usbdev.hjson

**Pull request merged**